TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/datahandlers/NettyPacketToHttpConsumer.java

```diff
@@ -220,12 +222,21 @@ public DiagnosticTrackableCompletableFuture<String,Void> consumeBytes(ByteBuf pa
                     } else {
                         log.atInfo().setMessage(()->"Signaling previously returned CompletableFuture packet write had an exception : "
                                 + packetData + " hash=" + System.identityHashCode(packetData)).log();
+                        metricsLogger.atError(cause)
+                                .addKeyValue("channelId", channel.id().asLongText())
+                                .addKeyValue("requestId", diagnosticLabel)
+                                .setMessage("Failed to write request packet").log();
                         completableFuture.future.completeExceptionally(cause);
                         channel.close();
                     }
                 });
         log.atTrace().setMessage(()->"Writing packet data=" + packetData +
                 ".  Created future for writing data="+completableFuture).log();
+        metricsLogger.atSuccess()
+                .addKeyValue("channelId", channel.id().asLongText())
+                .addKeyValue("requestId", diagnosticLabel)
+                .addKeyValue("sizeInBytes", readableBytes)
+                .setMessage("Request packet written to target").log();
```

Comment:
We don't have a guarantee that we've written a packet (the next level down the System stack could be buffering them and combining them).  Since these metrics are the most permanent things that we've done so far, it might be more appropriate to call these 'pieces' or something less network specific.